pandas/tests/window/moments/test_moments_expanding.py

```diff
@@ -418,3 +404,44 @@ def test_expanding_consistency(self, min_periods):
                     # GH 9422
                     if name in ["sum", "prod"]:
                         tm.assert_equal(expanding_f_result, expanding_apply_f_result)
+
+    @pytest.mark.parametrize("min_periods", [0, 1, 2, 3, 4])
```

Comment:
should these get `@pytest.mark.slow`?  `test_rolling_consistency_cov` clocks in at 38 seconds for me